### DISCUSSION TITLE
Why there is different in numbers on the speed test with different providers?

### DISCUSSION THREAD

**Discussion by balajeek** [alexjustesen/speedtest-tracker#689] [Q&A]
Please help me understand, how to read the numbers properly.
When it says Mbps, is it "Mega Bytes per second" or "Mega Bits per second", how to know which one they use:

I did a speed test one after another:
on Speedtest.net i got 19mbps down/ 28mbps up
on Cloudflare speed test i get 23mbps down/ 19mbps up
on this speedtest-tracker i get 82mbps down/ 23mbps up

Even if the servers are different, would it be that big a difference, how do i know what exactly is my ISP speed i am getting?

**Comment by alexjustesen** [comment]
`mbps` is megabits per second in this context. If you're seeing drastic performance changes this tool is meant to help you track those down. You can leave Speedtest Tracker running for a week or two and see if any trends come out of the data.
